htsget_s3_ingest.py

```diff
@@ -40,81 +48,157 @@ def collect_samples_for_genomic_id(genomic_id, client, prefix=""):
                 )
     return samples
 
-
-def main():
-    parser = argparse.ArgumentParser(description="A script that ingests a sample vcf and its index into htsget.")
-
-    parser.add_argument("--genomic_id", help="genomic sample id", required=False)
-    parser.add_argument("--clinical_id", help="clinical sample registration id", required=False)
-    parser.add_argument("--samplefile", help="file with list of genomic sample ids, optionally tab-delimited with clinical sample ids", required=False)
-    parser.add_argument("--endpoint", help="s3 endpoint")
-    parser.add_argument("--bucket", help="s3 bucket name")
-    parser.add_argument("--dataset", help="dataset name")
-    parser.add_argument("--awsfile", help="s3 credentials", required=False)
-    parser.add_argument("--access", help="access key", required=False)
-    parser.add_argument("--secret", help="secret key", required=False)
-    parser.add_argument("--region", help="optional: s3 region", required=False)
-    parser.add_argument("--prefix", help="optional: s3 prefix", required=False, default="")
-    parser.add_argument("--reference", help="optional: reference genome, either hg37 or hg38", required=False, default="hg38")
-    parser.add_argument('--indexing', action="store_true", help="optional: force re-indexing")
-
-    args = parser.parse_args()
-
+def htsget_ingest_from_s3(endpoint, bucket, dataset, token, genomic_id=None, clinical_id=None,
```

Comment:
Can we have better naming? because it could be from minio, not just s3

Reply:
Or we could have 2 separate functions depending on the call parameters

Reply:
Renamed in [58c0473](https://github.com/CanDIG/candigv2-ingest/pull/18/commits/58c04735eeaddba1b3cdc841f5c1a1dd372c7fc4)

Reply:
> Can we have better naming? because it could be from minio, not just s3

Minio is S3, for all intents and purposes: S3 is a more generic term.